src/containers/teleport/components/tabList/TabList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useNavigate } from 'react-router-dom';
+
+import ButtonTeleport from 'src/containers/warp/components/buttonGroup/indexBtn';
+import { TypePages } from '../../type';
+import s from './TabList.module.scss';
+
+type TabListProps = {
+  selected: TypePages;
+};
+
+function TabList({ selected }: TabListProps) {
+  const navigate = useNavigate();
```

Comment:
this also should be moved out of component scope.
I suggest to create TabList and TabButton, then container that reuse that components + BL. like TeleportTabList